src/panes.ts

```diff
@@ -43,12 +43,14 @@ export type SearchAndSelectPane = Pane<
   "search_and_select_pane",
   {
     title: string
+    description?: string
     options: Array<{
       label: string
+      sublabel?: string
       value: string
       image_url?: string
     }>
-    can_select_multiple?: boolean
+    device_selection_mode?: string
```

Comment:
This pane is used for things besides device selection, so let's do something like:

```suggestion
    selection_mode?: "MULTIPLE" | "SINGLE"
```

Reply:
I would also recommend lowercase, that seems to be our current public api convention e.g. `august`, `smartthings`, `none` | `single` | `multiple`

Reply:
Makes sense. Thanks!